index.html

```diff
@@ -388,30 +388,24 @@ <h3>Full Name</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+                  <img src="img/barbus-eric.jpg" alt="" class="avatar" />
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Bărbuș Eric</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    "Give a man a fish and you feed him for a day. Teach a man to catch a fish and you feed him for life." This motto perfectly represents our trainer, Cîrlugea Răzvan, for his enthusiastic way of initiating us in to the IT world as a Web Developer.
                   </p>
                   <p>
-                    Quidem, corrupti perspiciatis aliquid dicta cum odio
-                    sapiente tempora quas incidunt adipisci alias praesentium et
-                    excepturi error maxime aspernatur sint qui doloremque
-                    distinctio. Ex, unde.
+                    Regarding myself, I am a dynamic, open-minded and hard-working person. My goals are: learning, reinforcement and performance improvement. Coming from a performance sports area, my opinion is that one's succes comes from the group effort and team-work: "An succesful team is a group of many hands and one mind."
                   </p>
                 </div>
                 <div class="contact">
-                  <a href="#"
+                  <a href="https://www.linkedin.com/in/eric-barbus-a00b14236/"
```

Comment:
Have you tried generating a custom link for LinkedIn? It would be easier to include it in your CV (e.g. `https://www.linkedin.com/in/eric-barbus/` instead of `https://www.linkedin.com/in/eric-barbus-a00b14236/`)